angular/projects/admin-nrpti/src/app/mines/mines-records-list/mines-records-list.component.html

```diff
@@ -50,6 +51,31 @@ <h1 id="name" class="m-0">{{ (mine && mine.name) || '-' }}</h1>
       [filters]="filters">
     </search-filter-template>
   </section>
+
+  <section *ngIf="collectionAddEditState">
```

Comment:
When you select a row, it doesn't seem to change anything... `collectionAddEditState` doesn't appear to be flagged... but if you change tabs, suddenly the button appears! Looks like we need a change detection here, or more likely, this line:
`public collectionAddEditState = this.storeService.getItem(StateIDs.collectionAddEdit);`
Is setting it's default incorrectly

Reply:
Further do Dylan's comment, I don't actually think the button should appear.  If I navigate to the Records page and select a few records the buttons doesn't appear (correct, I should be using the actions menu).  Then if I navigate to the Collections tab and back to the Records tab I see the button "Add to Collection." When I click it the records are added to a blank collection that doesn't exist. 

I think when selecting records first any action taken on them must come form the Actions menu.

Reply:
Yeah, fair point. I think this just speaks to the confusion on the screens around the expected behaviour.

Reply:
This should hopefully be fixed now.  
The issue was that I was allowing the state to get updated when you weren't already editing a collection, which isn't a workflow that makes sense.  So now, the records list page should not be able to set the state from scratch, and only if someone else has already created it for them (ie, the collections add/edit page).